### PR TITLE
Add Trading using ChatGPT

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: 2023-03-trading-using-chat-gpt.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+domain: blog.octobot.online
+--- 
+
+# Ask ChatGPT about the future of the market
+Wouldn'it it be great to know what [ChatGPT](https://chat.openai.com/) thinks about a particular market direction ? 
+
+![chatgpt-logo](https://raw.githubusercontent.com/Drakkar-Software/OctoBot-Blog/master/resources/images/trading-using-chat-gpt/ChatGPT-Logo.png)
+
+
+While ChatGPT can't directly answer question such as "Will BTC/USD go up tomorrow ?", it can answer many related questions including forecasting trends.
```
Comment (Herklos) [inline 2023-03-trading-using-chat-gpt.md:16]: :+1:

File: 2023-03-trading-using-chat-gpt.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- Automatically trade based only on ChatGPT's predictions
+- Trade combining ChatGPT's predictions with other evaluators
+- Get ChatGPT's view on the market at anytime from the web interface and market status and get notified on any change
+
+![telegram-chat](https://raw.githubusercontent.com/Drakkar-Software/OctoBot-Blog/master/resources/images/trading-using-chat-gpt/telegram.png)
+
+# How to use ChatGPT in your OctoBot
+As ChatGPT's automated calls is a paid feature from [openai.com](https://openai.com/), there are 2 ways to use it in OctoBot.
+
+## 1. ChatGPT for free on Cloud OctoBots
```
Comment (Herklos) [inline 2023-03-trading-using-chat-gpt.md]: free ?

Comment (GuillaumeDSM) [reply 2023-03-trading-using-chat-gpt.md]: I changed it to "ChatGPT with covered costs on Cloud OctoBots"

File: 2023-03-trading-using-chat-gpt.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+When enabled, it will automatically ask ChatGPT about is opinion on every traded pair, on each timeframe.
+
+The type of data given to ChatGPT can be configured: the value of the asset price or different kind of technical evaluator can be sent to ChatGPT for it to make its opinion.
+
+Once enabled, `GPTEvaluator` behaves like any other technical evaluator, this means that it can combined with others, used to trade, to create trading signals or to notify you when the situation changes. 
+
+You can therefore:
+- Automatically trade based only on ChatGPT's predictions
+- Trade combining ChatGPT's predictions with other evaluators
+- Get ChatGPT's view on the market at anytime from the web interface and market status and get notified on any change
```
Comment (Herklos) [inline 2023-03-trading-using-chat-gpt.md:33]: :+1:

**Review by Herklos — APPROVED** [review]
:100:

**Pull request merged**